fiftyone/utils/complex_yolo/zoo.py

```diff
@@ -7,34 +7,65 @@
 """
 import logging
 import os
-from pkg_resources import packaging
-import warnings
+
+import numpy as np
 
 import eta.core.web as etaw
-import eta.core.utils as etau
 
 import fiftyone as fo
+import fiftyone.core.media as fom
 import fiftyone.core.utils as fou
 import fiftyone.utils.torch as fout
 import fiftyone.zoo.models as fozm
 
 fou.ensure_torch()
-import math
-import numpy as np
-import open3d as o3d
 import torch
+from torch.autograd import Variable
 
-device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
-Tensor = (
-    torch.cuda.FloatTensor if torch.cuda.is_available() else torch.FloatTensor
-)
+o3d = fou.lazy_import("open3d", callback=lambda: fou.ensure_import("open3d"))
 
-from .model import build_model
+from .model import ComplexYOLOv3
 
 
 logger = logging.getLogger(__name__)
 
 
+def apply_model(
```

Comment:
I updated this method's signature and the implementation of `predict_all()` below to work on non-grouped collections, because in future user should be able to directly pass a point cloud-only collection with feature maps stored in a supporting field.